src/main/java/net/javadiscord/javabot/util/WebhookUtil.java

```diff
@@ -103,30 +97,43 @@ public static CompletableFuture<ReadonlyMessage> mirrorMessageToWebhook(@NotNull
 		}
 		message.addEmbeds(embeds.stream().map(e -> WebhookEmbedBuilder.fromJDA(e).build()).toList());
 		List<Attachment> attachments = originalMessage.getAttachments();
-		@SuppressWarnings("unchecked")
-		CompletableFuture<?>[] futures = new CompletableFuture<?>[attachments.size()];
+		@SuppressWarnings("unchecked") CompletableFuture<?>[] futures = new CompletableFuture<?>[attachments.size()];
 		for (int i = 0; i < attachments.size(); i++) {
 			Attachment attachment = attachments.get(i);
-			futures[i] = attachment.getProxy().download().thenAccept(
-					is -> message.addFile((attachment.isSpoiler() ? "SPOILER_" : "") + attachment.getFileName(), is));
+			futures[i] = attachment.getProxy().download().thenAccept(is -> message.addFile((attachment.isSpoiler() ? "SPOILER_" : "") + attachment.getFileName(), is));
 		}
-		return CompletableFuture.allOf(futures).thenCompose(unused -> sendMessage(client, message))
-				.whenComplete((result, err) -> {
-					client.close();
-					if( err != null) {
-						ExceptionLogger.capture(err, WebhookUtil.class.getSimpleName());
-					}
-				});
+		return CompletableFuture.allOf(futures).thenCompose(unused -> sendMessage(client, message)).whenComplete((result, err) -> {
```

Comment:
Don't convert chained calls to a single line when they are split over multiple lines for readability.